presto-main/src/main/java/com/facebook/presto/operator/OperatorContext.java

```diff
@@ -442,6 +442,7 @@ public OperatorStats getOperatorStats()
                 outputPositions.getTotalCount(),
 
                 succinctBytes(physicalWrittenDataSize.get()),
+                succinctBytes(spillContext.getSpilledBytes()),
```

Comment:
Let's move this down to be after the `peak` stats. Also, surround it with blank lines since it is a separate stat.